website/src/categories/components/menu/usage.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Menu usage
+tags: usage
+eleventyNavigation:
+  parent: Menu
+  key: MenuUsage
+---
+
+<section class="no-heading">
+
+<div class="ds-example" style="gap: 4rem;">
+
+<sl-menu-button position="bottom">
+  <span slot="button">Actions</span>
+  <sl-menu-item><sl-icon name="smile"></sl-icon>Profile</sl-menu-item>
+  <sl-menu-item><sl-icon name="far-gear"></sl-icon>Settings</sl-menu-item>
+  <sl-menu-item><sl-icon name="far-trash"></sl-icon>Remove</sl-menu-item>
+</sl-menu-button>
+
+<sl-menu-button position="bottom-end" aria-label="Show more options">
+  <sl-icon name="ellipsis" slot="button"></sl-icon>
+  <sl-menu-item>Update</sl-menu-item>
+  <sl-menu-item>Remove</sl-menu-item>
+</sl-menu-button>
+
+</div>
+
+<div class="ds-code">
+
+  ```html
+  <sl-menu-button>
+    <span slot="button">Actions</span>
+    <sl-menu-item>
+      <sl-icon name="smile"></sl-icon>
+      Profile
+    </sl-menu-item>
+    <sl-menu-item>
+      <sl-icon name="far-gear"></sl-icon>
+      Settings
+    </sl-menu-item>
+    <sl-menu-item>
+      <sl-icon name="far-trash"></sl-icon>
+      Remove
+    </sl-menu-item>
+  </sl-menu-button>
+  
+  <sl-menu-button aria-label="Show more options">
+    <sl-icon name="ellipsis" slot="button"></sl-icon>
+    <sl-menu-item>Update</sl-menu-item>
+    <sl-menu-item>Remove</sl-menu-item>
+  </sl-menu-button>
+  ```
+
+</div>
+
+</section>
+
+<section>
+
+## When to Use
+The following guidance describes when to use the Menu component.
+
+### Contextual Actions
+Some interface elements can have additional actions, and providing a menu can enhance efficiency and streamline workflow. For instance, options to view details, open related documents, or access settings, offering extra details and actions specific to the element.
+
+### Reducing Visual Clutter
+Keeping a clean interface with repetitive elements (like cards or table rows), but additional actions are still necessary, a menu fixes this scenario while providing access to secondary actions. For example, sorting a list of elements or changing the view of the page.
+
+</section>
+
+
+<section>
+
+## When not to Use
+The following guidance describes when not to use the Menu component.
+
+### Primary Navigation
+Menus should not be used for primary navigation in applications because they are hidden until triggered by a specific action, making them less discoverable for users. Basic actions should always be visible and accessible to ensure users can efficiently navigate to essential features without additional steps. Relying on menus for primary navigation can also create inconsistency in accessing core functions by requiring users to learn multiple ways to perform basic tasks.
+
+</section>
+
+
+<section>
+
+## Anatomy
+
+|Item|Name| Description | Optional|
+|-|-|-|-|
+|1|Button Menu|This is the trigger button to open the menu. |no|
```

Comment:
where do these numbers refer to? Should there be an image?

Reply:
I think it's just described like e.g. in a card or dialog documentation, right @arecuenco-dsgn ?

Reply:
When we add the images to the documentation, these numbers match the tag numbers in the anatomy diagrams.